### ISSUE TITLE
[FEATURE] Implement debug options in the trainer (quick run, gradient norm inspection)

### ISSUE THREAD

**Issue by banctilrobitaille** [banctilrobitaille/kerosene#22]
**Is your feature request related to a problem? Please describe.**
Right now it is hard to debug the validation loop cause there is no way of bypassing the training loop.

**Describe the solution you'd like**
Ideally some options like quick_run or gradient norm inspection could be interesting
